troubleshooting.md

```diff
@@ -143,3 +143,23 @@ If you are using a useradd command within a Dockerfile with a large UID/GID, it
 #### Solution
 
 If the entry in the Dockerfile looked like: RUN useradd -u 99999000 -g users newuser then add the `--log-no-init` parameter to change it to: `RUN useradd --log-no-init -u 99999000 -g users newuser`. This option tells useradd to stop creating the lastlog file.
+
+#### Symptom
+
+If you are running podman or buildah on a home directory that is mounte noexec,
+then they will fail. something like
+
+```
+podman run centos:7
+standard_init_linux.go:203: exec user process caused "permission denied"
+```
+
+#### Solution
+
+Since the administrator of the system setup your homedir to be noexec, you
+will not be allowed to execute containers from storage in your homedir. It is
+possible to work around this by manually specifying a container storage
+path that's not in a noexec mount. Copying /etc/containers/storage.conf to
+~/.config/containers/ (creating the directory if necessary), you can adjust
+the paths to graphroot to a directory that is not on a noexec mount, your user
+has read/write privileges on, and is not on a tmpfs.
```

Comment:
I'm having trouble parsing the last sentence.  Perhaps:
"Simply copy the file /etc/containers.storage.conf to ~/.config/containers/ (creating the directory if necessary) ensuring the specified directory is not on a noexec mount, your user has read/write privileges to it, and it is not on tmpfs."

Reply:
Reworded.